mysql/mysql.go

```diff
@@ -104,13 +104,13 @@ func (s *mysqlPublisher) Publish(contentType string, content []byte, config map[
 	}
 	var key, value string
 	for _, m := range metrics {
-		key = sliceToString(m.Namespace())
+		key = sliceToString(m.Namespace().Strings())
 		value, err = interfaceToString(m.Data())
 		if err != nil {
 			logger.Printf("Error: %v", err)
 			return err
 		}
-		_, err := insert.Exec(m.Timestamp(), m.Source(), key, value)
+		_, err := insert.Exec(m.Timestamp(), m.Tags()["source"], key, value)
```

Comment:
probably using core.STD_TAG_PLUGIN_RUNNING_ON representing "source" or a constant is better